editor/Cargo.toml

```
@@ -29,3 +31,6 @@ package = "graphite-graphene"
 
 [dev-dependencies]
 env_logger = "0.8.4"
+
+[features]
+js = ["getrandom/js"]
```

Does this work
```suggestion
[target.'cfg(target_arch = "wasm")'.dependencies]
getrandom = { version = "0.2.3", features = ["js"] }
```
? I think you will need to add `resolver = 2` to the `[workspace]` section of the root `Cargo.toml` for this to work.

---

This seems to work ok. It forwards the js feature to getrandom. Documented here: https://doc.rust-lang.org/cargo/reference/features.html. Let me know if I am incorrect since I don't have a lot of experience with cargo features.

---

We ended up removing the dep entirely.